advisor/backend.go

```diff
@@ -98,23 +98,37 @@ func Create() (CommandDB, error) {
 	return t, nil
 }
 
-func (t *writer) AddSnap(snapName, summary string, commands []string) error {
-	bname := []byte(snapName)
-
+func (t *writer) AddSnap(snapName, version, summary string, commands []string) error {
 	for _, cmd := range commands {
+		var sil []Package
+
 		bcmd := []byte(cmd)
 		row := t.cmdBucket.Get(bcmd)
-		if row == nil {
-			row = bname
-		} else {
-			row = append(append(row, ','), bname...)
+		if row != nil {
+			if err := json.Unmarshal(row, &sil); err != nil {
+				return err
+			}
+		}
+		sil = append(sil, Package{Snap: snapName, Version: version})
```

Comment:
No summary?

Reply:
Indeed, I should add a comment. For the mapping of command->snap we do not need the summary, nothing is using that. But that is confusing so a comment is needed here.